flashcards/backend/docs/manage.rst

```diff
@@ -0,0 +1,7 @@
+manage module
+=============
```

Comment:
Django has it won documentation we do not need do document it, it is not our place to do it 😅

Reply:
Done.